ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -839,17 +823,23 @@ let on_notification server (notification : Client_notification.t) :
     Fiber.return state
   | Unknown_notification req -> (
     match req.method_ with
-    | "$/setTraceNotification"
+    | "$/setTraceNotification" -> (
```

Comment:
Why isn't setTrace parsed as a known method ?
I mean it might not be necessary to create a whole module just for that..

Reply:
Mostly because it's not part of the standard. I suppose this shouldn't really matter to use and we should make it easy for users to make use of this common method.

Reply:
Ah, sorry I should have caught that.
It was added in 3.16 and named `$/setTrace` (not `$/setTraceNotification`) : https://microsoft.github.io/language-server-protocol/specification#setTrace